### ISSUE TITLE
Adding setting with different default crashes, once

### ISSUE THREAD

**Issue by wiebeytec** [victronenergy/localsettings#1]
If I call AddSetting with `dbus-send`, it crashes with `UnboundLocalError` when I do so with a different value, but only the first time:

```
dbus-send --system --print-reply --dest=com.victronenergy.settings /Settings com.victronenergy.Settings.AddSetting string:"Gui" string:"Language" "variant:string:en" string:"s" variant:int32:0 variant:int32:0
method return sender=:1.26 -> dest=:1.763 reply_serial=2
   int32 0

# dbus-send --system --print-reply --dest=com.victronenergy.settings /Settings com.victronenergy.Settings.AddSetting string:"Gui" string:"Language" "variant:string:en" string:"s" variant:int32:0 variant:int32:0
method return se# dbus-send --system --print-reply --dest=com.victronenergy.settings /Settings com.victronenergy.Settings.AddSetting string:"Gui" string:"Language" "variant:string:en" string:"s" variant:int32:0 variant:int32:0
method return sender=:1.26 -> dest=:1.418 reply_serial=2
   int32 0nder=:1.26 -> dest=:1.418 reply_serial=2
   int32 0

# dbus-send --system --print-reply --dest=com.victronenergy.settings /Settings com.victronenergy.Settings.AddSetting string:"Gui" string:"Language" "variant:string:somethingelse" string:"s" variant:int32:0 variant:int32:0
Error org.freedesktop.DBus.Python.UnboundLocalError: Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/dbus/service.py", line 707, in _message_cb
    retval = candidate_method(self, *args, **keywords)
  File "/opt/color-control/localsettings/localsettings.py", line 380, in AddSetting
    myDbusObject._setValue(value, printLog=False)
UnboundLocalError: local variable 'myDbusObject' referenced before assignment

# dbus-send --system --print-reply --dest=com.victronenergy.settings /Settings com.victronenergy.Settings.AddSetting string:"Gui" string:"Language" "variant:string:somethingelse" string:"s" variant:int32:0 variant:int32:0
method return sender=:1.26 -> dest=:1.420 reply_serial=2
   int32 0
```

Looking at `/conf/settings.xml`, the default value does not seem effected.

This apparently does not have any adverse effects in Venus though? Settings are added all the time.

**Comment by mpvader** [comment]
Re-adding a setting with a different default should update the default to the new one. Same for min and max. It should not touch the value though.

**Comment by mpvader** [comment]
Closed with 75040490bf1deb94b7b9a8e03a2958bbd69b0bc9, in v1.06
